modules/nextflow/src/main/groovy/nextflow/cloud/aws/batch/S3Helper.groovy

```diff
@@ -43,6 +43,40 @@ class S3Helper {
             unset IFS
         }
         
+        nxf_retry() {
+            local max_attempts=\${ATTEMPTS-4}
+            local timeout=\${TIMEOUT-1}
```

Comment:
The values for these settings should be provided by: 
  * `aws.batch.maxTransferAttempts` 
  * `aws.batch.maxTransferTime` (or transferTimeout?)

Reply:
I would maybe name it `aws.batch.sleepBetweenAttempts` instead of `aws.batch.maxTransferTime` if you agree.

Reply:
What about instead `aws.batch.delayBetweenAttempts` ?

Reply:
👍